content/blog/understanding-the-dom/index.md

```diff
@@ -0,0 +1,519 @@
+	<div>Apples</div>
+	<div>Oranges</div>
+</div>
+```
+
+In this example, your browser only knows that you're looking to display text on-screen. If someone utilizing a screen reader reaches the site, the browser doesn't know that it should inform them that there are three items in a list (something that people with low vision would greatly value to know, in order to tab through the list effectively) as you've done nothing to inform the user that it is a list of items: only that it's a set of `<div>` generic containers.
+
+Likewise, when Google's robots walk through your site, they won't be able to parse that you're displaying lists to your users. As a result, your search rating for "list of best places" might be impacted since the site doesn't appear to contain any list at all.
+
+What can be done to remediate this? Well, by utilizing the proper tags, of course!
+
+```html
+<ol>
+	<li>Bananas</li>
+	<li>Apples</li>
+	<li>Oranges</li>
+</ol>
+```
+
+In this example, both the browsers as well as Google's scraper bots are able to discern that this is a list with three list items within it.
+
+> While there ARE tags that may potentially  impact SEO somewhat significantly, it's unlikely `<ul>` and `<li>` would significantly impact your SEO scores.
+>
+> Needless to say, it's still good to use symantic (correctly tagged) HTML as there are users that use screen-readers that will benefit greatly from these minor changes. Additionally, it can make code more readable and parsable with automated tools
+
+We're able to even add further metadata to an element by using attributes. For example, let's say that I want to add a title to the list to be read upon a screen reader gaining focus on the element, we could use the `aria-label` attribute:
+
+```html
+<ol aria-label="My favorite fruits">
+	<li>Bananas</li>
+	<li>Apples</li>
+	<li>Oranges</li>
+</ol>
+```
+
+In fact, the default metadata that is defaulted by specific tags can be directly applied manually to an element of a different tag. The metadata that is passed to the browser when using `<li>` is typically involving that element pertaining to a listitem, using the `role` attribute, we can add that information to a `<div>` itself.
+
+```html
+<ol>
+	<div role="listitem">Bananas</div>
+	<div role="listitem">Apples</div>
+	<div role="listitem">Oranges</div>
+</ol>
+```
+
+> It's worth mentioning that this example is generally considered malpractice. While you may have been able to preserve _some_ of the metadata from an `<li>` tag in a `<div>` element, it's extremely difficult to catch all of the defaults a browser might apply to the original tag that may enhanse a sight-impaired user's experience using screen-readers.
+>
+> This is all to say: unless you have a **really** good reason for using `role` rather than an approprate tag, stick with the related tag. Just as any other form of engineering, properly employing HTML requires nuance and logic to be deployed at the hand of the implementing developer.
```

Comment:
```suggestion
> This is all to say: unless you have a **really** good reason for using `role` rather than an appropriate tag, stick with the related tag. Just as any other form of engineering, properly employing HTML requires nuance and logic to be deployed at the hand of the implementing developer.
```